README.md

```diff
@@ -150,7 +184,7 @@ public class HomeActivity extends Activity {
 }
 ```
 
-Then just start `LockActivity` from inside your `Activity`.
+And start `LockActivity` from inside your `Activity`.
```

Comment:
It looks strange in English to start a sentence with "And" So I would juts use "Start"